api/config.py

```diff
@@ -74,12 +74,13 @@ class Config(object):
     # You can disable NRO updates for Name Requests by setting the variable in your .env / OpenShift configuration
     DISABLE_NAMEREQUEST_NRO_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_NRO_UPDATES', 0))
     DISABLE_NAMEREQUEST_SOLR_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_SOLR_UPDATES', 0))
+    DISABLE_NAMEREQUEST_NATS_UPDATES = int(os.getenv('DISABLE_NAMEREQUEST_NATS_UPDATES', 0))
```

Comment:
What is this change for? I didn't see it mentioned in the PR description header.

Reply:
Please see: https://github.com/bcgov/namex/pull/1151#discussion_r683610058

Reply:
This conversation can be closed. (I do not have permission to do so.)